README.Rmd

```diff
@@ -69,14 +69,14 @@ pak::pak("duckdblabs/duckplyr")
 
 ```{r attach}
 library(conflicted)
-library(duckplyr)
+library(dplyr)
 conflict_prefer("filter", "duckplyr")
 ```
 
 There are two ways to use duckplyr.
 
-1. To enable duckplyr for individual data frames, use `as_duckplyr_df()` as the first step in your pipe.
-1. To enable duckplyr for the entire session, use `methods_overwrite()`.
+1. To enable duckplyr for individual data frames, use `duckplyr::as_duckplyr_df()` as the first step in your pipe, without attaching the package.
+1. By calling `library(duckplyr)`, it is enabled for the entire session. To turn this off, call `methods_restore()`.
```

Comment:
```suggestion
1. By calling `library(duckplyr)`, it overwrites dplyr methods and is automatically enabled for the entire session without having to call `as_duckplyr_df()`. To turn this off, call `methods_restore()`.
```

I think we need to be very clear about the fact that this _overwrites_ (that's the key word to me)